assets/dev/js/editor/regions/panel/commands/change-device-mode.js

```diff
@@ -1,6 +1,6 @@
-import CommandBase from 'elementor-api/modules/command-base';
+import Command from 'elementor-api/modules/command';
```

Comment:
Replace back to command-base. and rename the command to command-core

Reply:
Handled.